common/src/main/java/io/netty/util/concurrent/DefaultThreadFactory.java

```diff
@@ -84,6 +83,10 @@ private static String toPoolName(Class<?> poolType) {
         }
     }
 
+    public DefaultThreadFactory(String poolName, boolean daemon, int priority) {
+        this(poolName, daemon, priority, null);
```

Comment:
You're reviewing code that was written before that comment was made. I pushed e30ad5114c37c6f0a0e31137b62b73b4cfa03d6d.